packages/venia-concept/src/components/PurchaseHistoryPage/PurchaseHistory/constants.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+export const PURCHASE_HISTORY_ITEM_PROP_TYPES = {
+    id: PropTypes.number,
+    imageSrc: PropTypes.string,
+    title: PropTypes.string,
+    date: PropTypes.date,
+    url: PropTypes.string
+};
```

Comment:
Let's keep these with the component definitions. It helps understand the component at a glance.